File: 01 Introdução/1079 Médias Ponderadas/1079.java
import java.util.Scanner;

class Main {
  public static void main(String str[]) {
    Scanner sc = new Scanner(System.in);
    int x = sc.nextInt();
    for (int i = 0; i < x; i++) {
      double num1 = sc.nextDouble();
      double num2 = sc.nextDouble();
      double num3 = sc.nextDouble();

      System.out.printf("%.1f\n", (num1 * 2 + num2 * 3 + num3 * 5) / 10);
    }

    sc.close();
  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1079
// Ranking 06044º, Time 0.471, Submiss Date 13/02/2023 15:13:37 (Java 14)

File: 01 Introdução/1079 Médias Ponderadas/1079.py
cases = int(input())
for i in range(0, cases):
    A, B, C = [float(x) for x in input().split(' ')]
    print(f"{(2*A + 3*B + 5*C) / 10:.1f}")

# https://www.beecrowd.com.br/judge/pt/problems/view/1079
# Ranking 01262º, Time 0.004, Submiss Date 05/03/2023 01:29:36 (Python 3.9)
